pacman-nodejs-app/deployments/mongo-deployment.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    name: mongo
+  name: mongo
+spec:
+  replicas: 0
```

Comment:
To reduce some duplication between the examples, I configured both deployments with 0 replicas, that way each example could issue the correct --scale operation for the proper size.